ppr-ui/src/components/registration/RegistrationBarTypeAheadList.vue

```diff
@@ -47,7 +47,7 @@
         </template>
         <template v-else>
           <v-list-item-content class="py-3 registration-list">
-            <span class="registration-list-item" v-html="`${parent.genFilteredText(item.text)}`"></span>
+            <span class="registration-list-item" v-html="item.text"></span>
```

Comment:
Awesome thanks, will need to remove `parent` from the slot scope above as well, as its no longer needed i think.